README.md

```diff
@@ -1,70 +1,51 @@
-# Getting Started with Create React App
+#	PROJECT: React Calculator
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+> This is a Calculator web app for Math Magicians online learning. Built using React.
+
+![Page](Assets/Images/home.png)
 
-In the project directory, you can run:
+## Available Scripts
 
 ### `npm start`
 
 Runs the app in the development mode.\
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
 
 ### `npm test`
```

Comment:
You should also add the steps that need to be taken before running `npm start`

Example:
> git clone 
> cd into folder